scripts/generateS2IconIndex.js

```diff
@@ -80,7 +81,7 @@ function generate(dir) {
 
     let imports = [typeImport];
     for (let size in sizes) {
-      imports.push(`import ${importName}_${size} from './${sizes[size]}';`);
+      imports.push(`import ${importName}_${size} from '${isIllustration ? 'illustration:' : ''}./${sizes[size]}';`);
     }
```

Comment:
Do we want to include this script to support multi-size icons/illustrations in the CLI as well?

Reply:
I assume there is a pattern we could use to detect this?